src/software/backend/robot_status.h

```diff
@@ -3,6 +3,7 @@
 #include <string>
 #include <vector>
 
+// TODO remove this file when radio moves to protobuf
```

Comment:
Can we quickly file a ticket for this? I forsee it getting totally forgotten.

Reply:
ive linked the ticket here